elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -6,19 +6,37 @@
 
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
-    pl.check_attribs(element, required_attribs=[], optional_attribs=['digits'])
+    required_attribs = []
+    optional_attribs = ['digits', 'display-matlab-tab', 'display-mathematica-tab', 'display-python-tab']
+    pl.check_attribs(element, required_attribs, optional_attribs)
 
 
 def render(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     digits = pl.get_integer_attrib(element, 'digits', 2)
+    display_matlab_tab = pl.get_boolean_attrib(element, 'display-matlab-tab', True)
+    display_mathematica_tab = pl.get_boolean_attrib(element, 'display-mathematica-tab', False)
+    display_python_tab = pl.get_boolean_attrib(element, 'display-python-tab', True)
+
+    # Default tab is Matlab
+    active_tab_matlab = True
+    active_tab_mathematica = False
+    active_tab_python = False
+    # If Matlab Display is False, will cycle through to next displayed tab
+    if display_matlab_tab is False:
+        active_tab_matlab = False
+        if display_mathematica_tab is True:
+            active_tab_mathematica = True
+        elif display_python_tab is True:
+            active_tab_python = True
```

Comment:
I think we should add a parameter to allow explicit specification of the default tab. Maybe `default-tab="matlab"`?

Reply:
I added this, I still hate how long and kind of "hard-coded" this is. Open to suggestions for better implementation strategies